backend/backend/events.py

```diff
@@ -131,7 +132,8 @@ def start_sse_stream(output_stream=sys.stdout):
 
         # Call function to check if any games in the database have a "playing"
```

Comment:
Can you update this comment to emphasise that we’re only checking one game’s state?